pkg/arvo/app/file-server.hoon

```diff
@@ -216,9 +216,10 @@
         =/  file  (as-octs:mimes:html .^(@ %cx scry-path))
```

Comment:
@tylershuster, you're scrying for all the files as `@`, and then later scrying again for `%woff2` as `$octs`. In the case of a `%woff2` mark, the first one will fail and you'll never get to the second one.

Reply:
Thank you, that's very clear.